src/extension/background.js

```diff
@@ -101,12 +101,37 @@ async function getProxyUrl({ id, url: tabUrl }) {
   }
 }
 
+/**
+ * Tries to guess if the current tab is a Franklin site.
+ * @param {Object} The tab
+ * @returns {Promise<Boolean>} True if the provided tab is a Franklin site
+ */
+async function guessIfFranklinSite({ id }) {
+  return new Promise((resolve) => {
+    chrome.scripting.executeScript({
+      target: { tabId: id },
+      func: () => {
+        const isHelixSite = document.head.querySelectorAll('script[src*="scripts.js"]').length > 0
```

Comment:
Franklin or Helix? 😉 
I'd recommend to consistently use franklin (or avoid branding altogether).